sources/documents/2_RTB/Verbali/Esterni/VerbaleEsterno_2024-05-06/local.tex

```diff
@@ -0,0 +1,26 @@
+%%%%%%%%%%%%%%
+%  COSTANTI  %
+%%%%%%%%%%%%%%
+
+% In questa prima parte vanno definite le 'costanti' utilizzate soltanto da questo documento.
+% Devono iniziare con una lettera maiuscola per distinguersi dalle funzioni.
+
+\newcommand{\DocTitle}{Verbale Riunione 2024-05-06}
+\newcommand{\DocVersion}{0.0.1}
+\newcommand{\DocApprovazione}{\raul}
+\newcommand{\DocUso}{Esterno}
+\newcommand{\DocDistribuzione}{
+	\Committente{} \\
+	Gruppo \GroupName{}
+}
+
+% La descrizione del documento
+\newcommand{\DocDescription}{
+	Questo verbale riporta gli aggiornamenti riguardanti lo studio e testing delle tecnologie che il team intende adottare, assieme ad alcune domande poste alla \glossario{Proponente} riguardo lo sviluppo dell'applicativo in direzione della produzione del \glossario{PoC}.   
```

Comment:
Questo verbale riporta gli aggiornamenti riguardanti lo studio e il collaudo delle tecnologie che il team intende adottare, assieme ad alcune domande poste alla \glossario{Proponente} relativamente allo sviluppo dell'applicativo, in vista della realizzazione del \glossario{PoC}.